daemon/cmd/daemon_main.go

```diff
@@ -1386,6 +1386,22 @@ func initEnv(vp *viper.Viper) {
 		log.Warn("IPSec encrypted overlay is enabled but IPSec is not. Ignoring option.")
 	}
 
+	// AWS does not support dualstack: https://docs.aws.amazon.com/eks/latest/userguide/cni-ipv6.html
```

Comment:
nit: should this "AWS does not support" or "EKS does not support"?

Reply:
(I definitely don't understand the whole configuration space here, so feel free to tell me I'm wrong).